app/resources/forms.js.template

```diff
@@ -368,12 +369,18 @@ function make_form_expandable(expandedSection) {
   });
 }
 
-function form_confirmation() {
-   var personName = $('given_name').value + ' ' + $('family_name').value;
-   if (confirm('Do you want to create a record for ' + personName + '?\n' +
-               'Press OK to submit the form.\n' +
-               'Press Cancel to return to the form.'))
-      return validate_fields();
-   else
-      return false;
+// Ask for confirmation when submitting a form
+// familyNameFirst: true or false
+function confirm_submit(familyNameFirst) {
+  var personName = $('given_name').value + ' ' + $('family_name').value;
+
+  if (familyNameFirst)
+    personName = $('family_name').value + ' ' + $('given_name').value;
+
+  if (validate_fields())
+    return (confirm('{% trans "Do you want to create a record for the following person?" %}'
```

Comment:
I should have pointed this earlier, but I feel that it's better if the message looks more like a confirmation e.g., "Are you sure to submit a record for the following person?"

What do you think?

Current message may be interpreted to create something new, independent of the record the user has just input.